src/__test__/components/data-exploration/CellInfo.test.jsx

```diff
@@ -81,9 +57,7 @@ describe('CellInfo', () => {
     };
 
     const component = mount(
-      <Provider store={store}>
-        <CellInfo componentType='heatmap' coordinates={coordinates} />
-      </Provider>,
+      <CellInfo componentType='heatmap' coordinates={coordinates} cellInfo={{}} />,
     );
 
     expect(component.find(Card).length).toEqual(0);
```

Comment:
Instead of doing this check, which only checks for the existence of the component, I think it would be better to check for the content of `cellInfo`, i.e. are all the information we are expecting available? It would be much easier to use react-testing-library then instead of enzyme.